projects/graph/src/graph.py

```diff
@@ -6,4 +6,14 @@
 class Graph:
     """Represent a graph as a dictionary of vertices mapping labels to edges."""
     def __init__(self):
-        pass  # TODO
+        self.vertices = {
+            0: {1, 2},
+            1: {1},
+            2: {0}
+        }
```

Comment:
If you ever need to initialize an empty set, the syntax is `set()`. You have to use that, because `{}` is reserved for empty dictionaries.